v2/tests/common.go

```diff
@@ -1,235 +0,0 @@
-	"time"
-
-	"github.com/brigadecore/brigade/sdk/v2"
-	"github.com/brigadecore/brigade/sdk/v2/core"
-	"github.com/stretchr/testify/require"
-)
-
-const (
-	testJobName = "test-job"
-	testTimeout = 120 * time.Second
-)
-
-var (
-	DefaultEventSubscriptions = []core.EventSubscription{
-		{
-			Source: "brigade.sh/cli",
-			Types: []string{
-				"exec",
-			},
-		},
-	}
-
-	DefaultConfigFiles = map[string]string{
-		"brigade.ts": fmt.Sprintf(`
-		import { events, Job } from "@brigadecore/brigadier"
-
-		events.on("brigade.sh/cli", "exec", async event => {
-			let job = new Job("%s", "alpine", event)
-			job.primaryContainer.sourceMountPath = "/var/vcs"
-			job.primaryContainer.command = ["ls"]
-			job.primaryContainer.arguments = ["-haltr", "/var/vcs"]
-			await job.run()
-		})
-
-		events.process()
-	`, testJobName)}
-)
-
-func GetRequiredEnvVar(t *testing.T, name string) string {
-	val := os.Getenv(name)
-	if val == "" {
-		t.Fatalf(
-			"value not found for required environment variable %s",
-			name,
-		)
-	}
-	return val
-}
-
-func assertWorkerPhase(
-	t *testing.T,
-	ctx context.Context, // nolint: revive
-	client sdk.APIClient,
-	e core.Event,
-	wantPhase core.WorkerPhase,
-) {
-	statusCh, errCh, err := client.Core().Events().Workers().WatchStatus(
-		ctx,
-		e.ID,
-	)
-	require.NoError(
-		t,
-		err,
-		"error encountered attempting to watch worker status",
-	)
-
-	timer := time.NewTimer(testTimeout)
-	defer timer.Stop()
-
-	for {
-		select {
-		case status := <-statusCh:
-			phase := status.Phase
-			if phase.IsTerminal() {
-				require.Equal(
-					t,
-					wantPhase,
-					phase,
-					"worker's terminal phase does not match expected",
-				)
-				return
-			}
-		case err := <-errCh:
-			t.Fatalf("error encountered watching worker status: %s", err)
-		case <-timer.C:
-			t.Fatal("timeout waiting for worker to reach a terminal phase")
-		}
-	}
-}
-
-func assertJobPhase(
-	t *testing.T,
-	ctx context.Context, // nolint: revive
-	client sdk.APIClient,
-	e core.Event,
-	job string, // nolint: unparam
-	wantPhase core.JobPhase,
-) {
-	statusCh, errCh, err := client.Core().Events().Workers().Jobs().WatchStatus(
-		ctx,
-		e.ID,
-		job,
-	)
-	require.NoError(t, err, "error encountered attempting to watch job status")
-
-	timer := time.NewTimer(testTimeout)
-	defer timer.Stop()
-
-	for {
-		select {
-		case status := <-statusCh:
-			phase := status.Phase
-			if phase.IsTerminal() {
-				require.Equal(
-					t,
-					wantPhase,
-					phase,
-					"job's terminal phase does not match expected",
-				)
-				return
-			}
-		case err := <-errCh:
-			t.Fatalf("error encountered watching job status: %s", err)
-		case <-timer.C:
-			t.Fatal("timeout waiting for job to reach a terminal phase")
-		}
-	}
-}
-
-func assertVCSLogs(
-	t *testing.T,
-	ctx context.Context, // nolint: revive
-	client sdk.APIClient,
-	e core.Event,
-	wantLogs string,
-) {
-	assertLogs(
-		t,
-		ctx,
-		client,
-		e,
-		&core.LogsSelector{Container: "vcs"},
-		wantLogs,
-	)
-}
```

Comment:
This function was called in only one or two places and it's not significantly more verbose to call `assertLogs()` directly with the correct `LogSelector`, so I've deleted this function.